src/main.rs

```diff
@@ -37,29 +37,34 @@ use alacritty::tty::{self, process_should_exit};
 use alacritty::util::fmt::Red;
 
 fn main() {
+    // Load command line options
+    let options = cli::Options::load();
 
     // Load configuration
-    let config = Config::load().unwrap_or_else(|err| {
+    // If a configuration file is given as a command line argument we don't generate a default file.
+    // If an empty configuration file is given, i.e. /dev/null, we load the compiled-in defaults.
+    let config_path = options.config.clone().unwrap_or_else(|| {
```

Comment:
Is there any way to do this without `clone()`? I cannot use `as_ref()` because then `Config::default_config()` also has to return a reference.